search_service/api/document.py

```diff
@@ -65,7 +65,11 @@ def post(self) -> Tuple[Any, int]:
 
         try:
             table_dict_list = [literal_eval(table_str) for table_str in args.get('data')]
-            data, errors = self.schema(many=True).load(table_dict_list)
+            try:
+                data, errors = self.schema(many=True).load(table_dict_list), None
+            except Exception as e:
+                errors = e.args
```

Comment:
should this be e.messages? https://marshmallow.readthedocs.io/en/stable/upgrading.html#schemas-are-always-strict

I also suggest excepting on ValidationError

Reply:
same question in a few other places

Reply:
also, style suggestion: Instead of assigning None to `errors` in line 69, pretty sure it'd be more clear to just put the logging directly into the `except`

Reply:
sounds good, added the changes